src/payment/strategies/masterpass/masterpass.ts

```diff
@@ -1,3 +1,5 @@
+export const masterpassCallbackUrl = '/checkout.php?action=set_external_checkout&provider=masterpass&gateway=stripe&';
```

Comment:
I would argue that `${window.location.origin}` can be included here and removed elsewhere 🍹

Reply:
I agree with @Sure-Round here. I'd prefer if we make this a function and return the whole thing. Also, that last `&` is a bit weird. The consumer needs to know this url finishes with a `&`.

Maybe something like 
```js
function getCallbackUrl(origin: string): string {
```